csharp/src/Ice/OutputStream.cs

```diff
@@ -2256,7 +2256,8 @@ internal void setEncoding(EncodingVersion encoding)
     //
     private EncodingVersion _encoding;
 
-    private bool isEncoding_1_0() => _encapsStack != null ? _encapsStack.encoding_1_0 : _encoding.Equals(Util.Encoding_1_0);
+    private bool isEncoding_1_0() => _encapsStack !=
+        null ? _encapsStack.encoding_1_0 : _encoding.Equals(Util.Encoding_1_0);
```

Comment:
This still looks and odd formatting. I will avoid putting the operands of the `!=` in different lines.